safe_transaction_service/account_abstraction/tests/test_views.py

```diff
@@ -438,3 +440,20 @@ def test_safe_operation_valid_until_create_view(
                 ]
             },
         )
+
+        # Fix `valid_until`, set it in the future
```

Comment:
Could you update the test doc? https://github.com/safe-global/safe-transaction-service/pull/1984/files#diff-3c6ffe3225ec603b1d607b99ecc7b1c05fe58b8828f77f8be255334c91b3c16bR381 
Because now is testing also the success case.